lms/djangoapps/courseware/views/index.py

```diff
@@ -503,7 +503,7 @@ def _create_courseware_context(self, request):
         )
 
         courseware_context['course_sock_fragment'] = CourseSockFragmentView().render_to_fragment(
-            request, course=self.course_overview)
+            request, course=self.course)
```

Comment:
Do you think this will slow down the loading of this page noticeably? Probably not, right?

Reply:
Well, I don't think this slows down the page at all. I checked that the `CourseSockFragment` don't really use anything that is uniquely provided by `self.course_overview`. Plus, we already have `self.course` loaded in this logic block